src/lib/crypto/kimchi_bindings/stubs/src/pasta_fp_plonk_verifier_index.rs

```diff
@@ -75,73 +95,100 @@ impl From<CamlPastaFpPlonkVerifierIndex> for VerifierIndex<Vesta> {
         let shifts: Vec<Fp> = shifts.iter().map(Into::into).collect();
         let shift: [Fp; PERMUTS] = shifts.try_into().expect("wrong size");
 
+        let lookup_index: std::option::Option<LookupVerifierIndex<Vesta>> =
+            index.lookup_index.map(Into::into);
+
+        // FIXME: Is the flag computation correct ?
+        // Should both xor fields be synced ?
```

Comment:
Good question. They are distinct because we used to also use xor lookups for chacha, but now that it's gone, there's not much point in both any more.
Since we've frozen the code though, I don't think we can change the proof type to remove it, sadly.